### PR TITLE
Adding Azure Demo and related Azure modules

### PR DESCRIPTION
**Pull request by Krishna-Vutukuri**
This change contains Azure Compute related demo files as well associated AzureRM modules. I've run the demo script using alpha.9 release of PowerShell on Ubuntun 14.04 and Windows 10. Thank You!

### CODE REVIEW

**Comment by TravisEz13** [comment]
Do we need to include the [Azure-PowerShell License](https://github.com/Azure/azure-powershell/blob/dev/LICENSE.txt) with the binaries?

**Comment by lzybkr** [comment]
Why binaries?  Can't we pull them in from somewhere?

**Comment by KrishnaV-MSFT** [comment]
@TravisEz13  @lzybkr you've raised a valid point and I agree with it. Let me explore alternatives. I could publish to PowerShellGallery but Install-Module is broken currently with -alpha.9 release. I will figure out a workaround.

**Comment by TravisEz13** [comment]
Based on comments waiting on author

**Comment by lzybkr** [comment]
@KrishnaV-MSFT - instead of a new commit, can you rewrite history so that the binary files were never introduced.  Squashing the commits should do it.

File: demos/Azure/readme.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Demo: Managing Azure using PowerShell
+
+This demo shows management of Azure Compute resource using Azure Resource Management (ARM) cmdlets.
+
+## Prerequisites ##
+- Install PowerShell 6.0.0-alpha.9
+- Copy AzureRM.NetCore.Preview, AzureRM.Profile.NetCore.Preview and AzureRM.Resources.NetCore.Preview modules to a local directory.
+   - These modules are located under the demos\Azure directory of the PowerShell repo.
```
Comment (lzybkr) [inline demos/Azure/readme.md]: Can you update this now that the modules are not in the PowerShell repo?

**Pull request merged**